charts/newrelic-infrastructure/values.yaml

```diff
@@ -142,39 +125,20 @@ ksm:
 controlPlane:
   # -- Deploy control plane monitoring component.
   enabled: true
-  # -- Run Control Plane scraper with `hostNetwork` even if `privileged` is set to false.
-  # `hostNetwork` is required for most control plane configurations, as they only accept connections from localhost.
-  unprivilegedHostNetwork: false
```

Comment:
I agree with getting rid of this flag, but we really need to decide what is going to happen with the controlPlane pods if `privileged: false`. Failing is not an option as this is something that can come from uninterative flows (guided install).

Reply:
I would even suggest leaving everything related to `privileged` out of this PR and discuss on a different one.